.gitignore

```diff
@@ -250,4 +250,21 @@ dmypy.json
 # Cython debug symbols
 cython_debug/
 
-# End of https://www.toptal.com/developers/gitignore/api/python,pycharm
\ No newline at end of file
+# End of https://www.toptal.com/developers/gitignore/api/python,pycharm
+
+
+# IDEs
+.idea
+.vscode
```

Comment:
Better to copy paste visual studio code example
https://www.toptal.com/developers/gitignore/api/visualstudiocode

Reply:
nice that you added an entry, but please remove `.vscode` as well